README.md

```diff
@@ -22,7 +22,7 @@ If you have been using go-cookbook previously then please note that:
 
 ### Java
 
-Please note that java (>= 7) is needed to run Go server and agents. This cookbook
+Please note that java 8 is recommended to run Go server and agents. This cookbook
```

Comment:
Java 7 is deprecated already and will not be supported in GoCD 17.

https://www.go.cd/releases/